test/fixture/postgres/osm.sql

```diff
@@ -73,5 +73,9 @@ CREATE TABLE users (
     interval_col interval,
     json_col json,
     date_col date,
-    unspported_path_type path
+    unspported_path_type path,
+    name_type_col name,
+    json_array_col json[],
+    jsonb_array_col jsonb[],
+    timetamptz_array_col timestamptz[]
```

Comment:
looks like you have a typo over here.
CI logs: https://travis-ci.org/SweetIQ/schemats/jobs/277855858#L1482

Reply:
Right you are, totally missed that! I'll change it now